packages/api/cms-api/src/generator/crud-generator.decorator.ts

```diff
@@ -1,6 +1,6 @@
 export interface CrudGeneratorOptions {
     targetDirectory: string;
-    requiredPermission?: string[] | string;
+    requiredPermission?: string[] | string | { read: string[]; create: string[]; update: string[]; delete: string[] };
```

Comment:
why not support `string[] | string` for read/create/etc as well?

Reply:
is updated. I thought I would need the isArray stuff again I got rid off. but because the permission-decorator now supports string and string[] this is not needed.